client/server/pages/test/index.js

```diff
@@ -24,6 +27,7 @@ jest.mock( 'calypso/server/bundler/utils', () => ( {
 } ) );
 
 jest.mock( 'calypso/sections', () => {
+	// eslint-disable-next-line no-shadow
```

Comment:
added this line because I moved the imports to the top which then triggered the eslint failure